app/views/logix/examples.html.erb

```diff
@@ -1,3 +1,14 @@
+<script>
+  $(window).scroll(function() {
+    if ($(document).scrollTop() > 70) {
+        $("nav").addClass('affix');
+        console.log("OK");
+    } else {
+        $("nav").removeClass('affix');
+    }
+});
+</script>
```

Comment:
You can add this in application layout if you want to add this in every pages

Reply:
![ezgif com-video-to-gif(3)](https://user-images.githubusercontent.com/47236824/76700412-23a45800-66dd-11ea-9278-3dc392d879d7.gif)